content/swiftui/concepts/views/terms/scrollview/scrollview.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'ScrollView'
+Description: 'Scrollable view within the scrollable content region.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'ScrollView'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`ScrollView`** display the content within the scrollable content region. Based on platform appropriate scroll gestures, or how the code works, the view can scroll vertically, horizontally or both.
+
+## Syntax
+
+```pseudo
+var body: some View {
+    ScrollView(.vertical) {
+        Subviews here
+    }
+}
+```
+
+```pseudo
+var body: some View {
+    ScrollView(.horizontal) {
+        Subviews here
+    }
+}
+```
+
+The `ScrollView` view rests within the body of a `View`.
+
+## Example
+
+In this example, a `ScrollView`, that will scroll horizontally, contains a `LazyHStack` that consists of columns stacked next to each other in the view:
```

Comment:
```suggestion
In the example below, the `ScrollView` scrolls horizontally and contains a `LazyHStack` that consists of columns stacked next to each other in the view:
```

Reply:
Can we also link the lazy HStack entry? link: https://www.codecademy.com/resources/docs/swiftui/views/lazyHStack